lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -37,6 +37,7 @@ class DecidirGateway < Gateway
         56 => STANDARD_ERROR_CODE[:card_declined],
         57 => STANDARD_ERROR_CODE[:card_declined],
         76 => STANDARD_ERROR_CODE[:call_issuer],
+        91 => STANDARD_ERROR_CODE[:invalid_number],
```

Comment:
Nice catch noticing `91` wasn't handled! You've thought about this more than me, but is `invalid_number` the right message for this? My Spanish is rusty, but it looks like error 91 translates roughly to:

```
OUTLINE LINE ISSUER

Request telephone authorization, if approved, load the code obtained and leave the operation in
```

Reply:
I think this might be a part of a discussion in the Jira ticket that Branden and I were having about error_code's, using the int versus a standardized string: https://spreedly.atlassian.net/secure/RapidBoard.jspa?rapidView=52&projectKey=CE&modal=detail&selectedIssue=CE-378&assignee=5e26fb3aef603b0ca6f6f25a

Seems like there is inconsistent use across the board as to whether or not we return the error code as we get it from the gateway or try to map it to something Spreedly understands. On one hand, it makes sense that our response is standardized, but on the other, that `91` error code maps to something internally understood by Decidir and as you highlighted, it has a much more specific meaning and probably holds more value than the standardized value we are mapping it to.

That being said, I'm not sure if I have enough information as to why we standardize in some gateways and why we don't in others.

Reply:
@naashton Ah okay. I was just wondering about this one since Decidir's error message didn't seem to match up to "invalid number". If it's part of another conversation I'll leave it alone here.